packages/docregistry/src/context.ts

```diff
@@ -130,6 +132,13 @@ class Context<T extends DocumentRegistry.IModel> implements DocumentRegistry.ICo
     return this._contentsModel;
   }
 
+  /**
+   * The ServiceManager associated with the document.
+   */
+  get manager(): ServiceManager.IManager {
```

Comment:
I'd rather expose `localPath` as a property on the context than expose the service manager.

Reply:
Good idea, that is indeed a simplification.